assets/css/style.css

```diff
@@ -86,17 +124,22 @@ p {
     background-color: #2589bd;
 }
 
-.benefit-lead {
```

Comment:
There is code duplication which can be generalised in to one css selector. Please identify the duplication and reduce the duplication accordingly.